src/imagetilesource.js

```diff
@@ -0,0 +1,274 @@
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+(function ($) {
+
+    /**
+     * @class ImageTileSource
+     * @classdesc The ImageTileSource allows simple image to be loaded
+     * into an OpenSeadragon Viewer.
+     *
+     * @memberof OpenSeadragon
+     * @extends OpenSeadragon.TileSource
+     * @param {Object} options Options object.
+     * @property {String} options.url URL of the image
+     * @property {Boolean} options.buildPyramid If set to true, a pyramid will
+     * be built internally to provide a better downsampling.
+     * @property {String|Boolean} options.crossOriginPolicy Valid values are
+     * 'Anonymous', 'use-credentials', and false. If false, image requests will
+     * not use CORS preventing internal pyramid building for images from other
+     * domains.
+     * @property {String|Boolean} options.ajaxWithCredentials Whether to set the
+     * withCredentials XHR flag for AJAX requests (when loading tile sources)
+     */
+    $.ImageTileSource = function (options) {
+
+        this.options = options;
+        $.TileSource.apply(this, [options]);
+
+    };
+
+    $.extend($.ImageTileSource.prototype, $.TileSource.prototype, /** @lends OpenSeadragon.ImageTileSource.prototype */{
+        /**
+         * Determine if the data and/or url imply the image service is supported by
+         * this tile source.
+         * @function
+         * @param {Object|Array} data
+         * @param {String} optional - url
+         */
+        supports: function (data, url) {
+            return data.type && data.type === "image";
+        },
+        /**
+         *
+         * @function
+         * @param {Object} options - the options
+         * @param {String} dataUrl - the url the image was retreived from, if any.
+         * @return {Object} options - A dictionary of keyword arguments sufficient
+         *      to configure this tile sources constructor.
+         */
+        configure: function (options, dataUrl) {
+            return options;
+        },
+        /**
+         * Responsible for retrieving, and caching the
+         * image metadata pertinent to this TileSources implementation.
+         * @function
+         * @param {String} url
+         * @throws {Error}
+         */
+        getImageInfo: function (url) {
+            var image = new Image();
+            var _this = this;
+
+            if (this.options.crossOriginPolicy) {
+                image.crossOrigin = this.options.crossOriginPolicy;
+            }
+            if (this.options.ajaxWithCredentials) {
+                image.useCredentials = this.options.ajaxWithCredentials;
+            }
+
+            image.addEventListener('load', function () {
```

Comment:
`addEventListener` doesn't work in IE8, which OSD still supports. That said, we can make the choice to exclude this new tile source from IE8 support. If we do, we should do so explicitly. At any rate, we have $.addEvent to manage the browser differences.